src/ui/Button/Button.js

```diff
@@ -18,13 +18,14 @@ const Button = styled.button`
       ? props.theme.colors[props.color][props.outlined || props.transparent ? 'main' : 'contrast']
       : 'inherit'};
   cursor: pointer;
-  font-family: ${props => props.theme.text.secondary};
+  font-family: ${props =>
+    props.variant ? props.theme.text[props.variant] : props.theme.text.secondary};
   ${props =>
     props.size
       ? responsive('font-size', 'size', value => `${value}px`)
       : 'font-size: calc(13 / 16 * 1rem)'};
-  font-weight: 800;
-  letter-spacing: 2px;
+  font-weight: ${props => (props.weight ? props.weight : 800)};
+  letter-spacing: ${props => `${props.spacing ? `${props.spacing}px` : '2px'}`};
```

Comment:
please use the `rem` helper from polished. `${rem(props.spacing})}` See elsewhere in the codebase for reference.

Reply:
Also, might as well add `rem('2px')` for the default too